Neos.Flow/Tests/Functional/Persistence/Fixtures/SuperEntityRepository.php

```diff
@@ -11,7 +11,7 @@
  * source code.
  */
 
-use Neos\Flow\Persistence\Repository;
+use Neos\Flow\Persistence\Doctrine\Repository;
```

Comment:
Why? Is the fixture specific for a test related to the Doctrine repository?

Reply:
Same here

Reply:
👍